### PR TITLE
Kotlin 1.2.20 with kapt build cache enabled

### PR DESCRIPTION
**Pull request by KeithYokoma**
Kotlin 1.2.20 release note:
https://blog.jetbrains.com/kotlin/2018/01/kotlin-1-2-20-is-out/

- Kotlin 1.2.20 released with kapt build cache!

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -81,6 +81,10 @@ android {
     }
 }
 
+kapt {
+    useBuildCache = true
```
Comment (konifar) [inline app/build.gradle]: 📝

**Comment by konifar** [comment]
Thanks! I'll try to introduce it to company app production code 😉

**Comment by konifar** [comment]
LGTM!

**Comment by takahirom** [comment]
@konifar  me too 😄

File: app/build.gradle (hunk 1 above)
Comment (kaelaela) [reply app/build.gradle]: 📝

**Comment by konifar** [comment]
If something will be wrong, let's revert then

**Pull request merged**